cineast-core/src/main/java/org/vitrivr/cineast/core/iiif/imageapi/v2/ImageInformation_v2.java

```diff
@@ -0,0 +1,431 @@
+package org.vitrivr.cineast.core.iiif.imageapi.v2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.vitrivr.cineast.core.data.Pair;
+import org.vitrivr.cineast.core.iiif.imageapi.ImageApiVersion;
+import org.vitrivr.cineast.core.iiif.imageapi.ImageApiVersion.IMAGE_API_VERSION;
+import org.vitrivr.cineast.core.iiif.imageapi.ImageInformation;
+
+/**
+ * ImageInformation object used to parse Image API 2.1.1 image information request's JSON response
+ *
+ * @author singaltanmay
+ * @version 1.0
+ * @created 29.05.21
+ */
+public class ImageInformation_v2 implements ImageInformation {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+  /**
+   * A list of profiles, indicated by either a URI or an object describing the features supported. The first entry in the list must be a compliance level URI.
+   */
+  @JsonProperty(required = true)
+  private List<Object> profile;
```

Comment:
Is `Object` really the most specific you can be here?

Reply:
`profile` key in version `2.1.1` and below can contain multiple different types of values which are not supported in Java and thus needs to be manually checked for type.